scripts/lib/aws/elasticache.sh

```diff
@@ -239,6 +307,59 @@ aws_wait_replication_group_deleted() {
   k8s_controller_reload_credentials "$service_name"
 }
 
+# aws_get_replication_group_json returns the JSON description of the replication group of interest
+# aws_get_replication_group_json requires 1 arguments:
+#    replication_group_id
+aws_get_replication_group_json() {
+  if [[ $# -ne 1 ]]; then
+    echo "FATAL: Wrong number of arguments passed to ${FUNCNAME[0]}"
+    echo "${FUNCNAME[0]} replication_group_id"
+    exit 1
+  fi
+  echo $(daws elasticache describe-replication-groups --replication-group-id "$1" | jq -r -e ".ReplicationGroups[0]")
```

Comment:
this may just be copy-pasta, but the `jq -e` switch causes the exit code to be 0 when the last output value is not  `false` or `null`, 1 otherwise. But you're not checking for the value of `$?` anywhere that I can see...

Reply:
There are tests where I check the exit code but you're right that it's copy-pasta, I'll go through and be more intentional about checking the exit code, or just removing -e where it's not relevant